docs/technical-details/travel-speeds/index.md

```diff
@@ -70,7 +70,7 @@ servers as it can lead to undesirable routes.
 ### Walking profiles
 The travel speeds for `foot-*` profiles (walking and hiking) are set to
 5 km/h on all allowed waytypes.
-For ways with a [`sac_scale`](../extra-info/Trail-Difficulty) higher than
+For ways with a [`sac_scale`](../../api-reference/endpoints/directions/extra-info/trail-difficulty.md) higher than
```

Comment:
probably use absolute paths. + mark docs folder as sources root (that's the root for vitepress) to make md links work

Reply:
This is an IntelliJ-related issue, no?

Reply:
intellij/vitepress. As `<reporoot>/docs` is defined as the root in vitepress. all links can be relative to that folder e.g.:
`[these instructions](/contributing/opening-project-in-intellij)` in `building-from-source.md`
They also work without the `.md` extension but intellij only finds the path with extension.
My question is if we should do this the vitepress or the intellij/filesystem way?

Reply:
https://vitepress.dev/guide/routing#linking-between-pages
suggests to not use endings but is fine with either relative or absolute (relative to vitepress base which is the docs folder).
Personal preference would be absolute, but file links definitely should work in intellij with strg + click.
This currently only works if the `docs` folder is marked as 'sources root' and the `.md` ending is put in the link